index.js

```diff
@@ -3,6 +3,7 @@ class LiteApi {
         this.apiKey = apiKey;
         this.serviceURL = "https://api.liteapi.travel/v3.0";
         this.bookServiceURL = "https://book.liteapi.travel/v3.0";
+        this.baseUrl = 'https://da.liteapi.travel';
```

Comment:
This would be "dashboardUrl", right?

Reply:
Yes, and it is the URL used to get vouchers

Reply:
My question here is to change this variable name to `dashboardURL`

Reply:
ok  i will change this variable name from baseUrl to dashboardURL